app/services/label/inventory_label.rb

```diff
@@ -0,0 +1,47 @@
+module Label
+  class InventoryLabel < BaseLabel
+    attr_accessor :inventory_number, :label, :dots, :width, :field_orientation, :height, :print_count
+
+    BARCODE_QR_URL = "https://redirect.crossroads.org.hk/inventory?num=".freeze
+
+    INVENTORY_LABEL_DOTS = 300
+
+    # The dimensions of the label
+    INVENTORY_LABEL_WIDTH    = 2
+    INVENTORY_LABEL_HEIGHT   = 1
+    INVENTORY_LABEL_FIELD_ORIENTATION = :landscape
+
+    def initialize(inventory_number, print_count)
```

Comment:
I think it might be better to use Rails `options` hash style arguments here as it allows more flexibility in the parameters that are set... some labels will set different arguments (e.g. inventory_number) and some will have different params. By using `options` you can have a unified argument across all label types whilst being flexible enough for each particular label's needs.

I'll branch off your branch here with some suggestions and you can merge in if you like it.

Reply:
One other benefit we often forget is that options tell us what it means without looking at the method's code

`InventoryLabel.new('123', 3)` -> we don't know what `3` stands for without opening up the file with the method's code
`InventoryLabel.new('123', print_count: 3)` -> we immediately know what it means

❤️ Ruby options ❤️